lib/src/browser/disposable_browser.dart

```diff
@@ -158,6 +161,11 @@ class Disposable implements disposable_common.Disposable {
   }
 
   @override
+  void flagLeak([String description]) {
```

Comment:
What I should have said is that the parent is always going to be `_InnerDisposable`, but we could use the description to provide better context to consumers.